programs/x509/cert_write.c

```diff
@@ -744,6 +807,22 @@ int main( int argc, char *argv[] )
         mbedtls_printf( " ok\n" );
     }
 
+    if( opt.ext_key_usage )
+    {
+        mbedtls_printf( "  . Adding the Extended Key Usage extension ..." );
+        fflush( stdout );
+
+        ret = mbedtls_x509write_crt_set_ext_key_usage( &crt, opt.ext_key_usage );
+        if( ret != 0 )
+        {
+            mbedtls_strerror( ret, buf, 1024 );
```

Comment:
(comment) should really be

```suggestion
            mbedtls_strerror( ret, buf, sizeof(buf) );
```

but this is how the pre-existing code is, so don't change it

Reply:
@daverodgman looks like you've changed the `1024` after this one, not this one :)